Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -612,6 +608,31 @@ protected theorem repr_reindex (b : OrthonormalBasis ι 𝕜 E) (e : ι ≃ ι')
 
 end OrthonormalBasis
 
+namespace EuclideanSpace
+
+variable (𝕜 ι)
+
+/-- The basis `Pi.basisFun` bundled as an orthormal basis of `EuclideanSpace 𝕜 ι`. -/
```

Comment:
```suggestion
/-- The basis `Pi.basisFun`, bundled as an orthormal basis of `EuclideanSpace 𝕜 ι`. -/
```
Can you add a lemma that relates this definition with [`PiLp.basisFun`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/NormedSpace/PiLp.html#PiLp.basisFun)?